pybamm/util.py

```diff
@@ -331,9 +331,9 @@ def install_jax(arguments=None):  # pragma: no cover
     """
     Install compatible versions of jax, jaxlib.
 
-    Command Line Interface:
-    -----------------------
-    >>> pybamm_install_jax
+    Command Line Interface::
+
+        $ pybamm_install_jax
 
     optional arguments:
```

Comment:
Looks great! Nitpicking a bit -

<img width="706" alt="image" src="https://user-images.githubusercontent.com/74055102/221401082-a9e03dc2-e929-4315-994f-db2af2b09c61.png">

Can the optional arguments render in separate lines?

Reply:
Yup @Saransh-cpp, the whole point of docs is to have easy readability so nitpicking will do us good. Will make the commits soon.